src/romaine/core.py

```diff
@@ -14,6 +15,7 @@ def __init__(self):
             Initialise Romaine core.
         """
         self.steps = {}
+        self.parser = Parser()
```

Comment:
I'm not entirely sure why the parser has to be attached as an attribute of the core - is it not something the core uses to get the parser's result?